src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,42 +1,92 @@
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import moment from 'moment';
+
 import { focusElement } from '../../../common/utils/helpers';
 import OMBInfo from '../../../common/components/OMBInfo';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
-
+import RequiredLoginView from '../../../common/components/RequiredLoginView';
 import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { submitIntentToFile } from '../../../common/schemaform/actions';
+
+import formConfig from '../config/form';
 
 class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      verify: false
+    };
+  }
+
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  handleLoadPrefill = () => {
+    // TODO: determine payload
+    submitIntentToFile('name/ssn/other', formConfig.intentToFileUrl, formConfig.trackingPrefix);
+    // TODO: store confirmation number in formData
+  }
+
   render() {
+    const { signingIn, requiredAccountLevel } = this.state;
+    const { user, loginUrl, verifyUrl } = this.props;
+    const savedForm = user.profile && user.profile.savedForms
+      .filter(f => moment.unix(f.metadata.expires_at).isAfter())
+      .find(f => f.form === this.props.formId);
+
     return (
       <div className="schemaform-intro">
-        <FormTitle title="This page needs attention"/>
+        <FormTitle title="Disability Claims for Increase"/>
         <p>Equal to VA Form 21-526EZ.</p>
-        <SaveInProgressIntro
+        {(!signingIn && !sessionStorage.userToken) && <SaveInProgressIntro
+          prefillAvailable={user.profile.accountType === 3}
```

Comment:
Is this temporary? I don't think this is the way to determine if they have pre-fill information available.

That does bring up a question we've been skirting around, though: Are we using a single consolidated `vets-api` endpoint to load all the pre-fill information? It would make the front end implementation much cleaner (and I think that's what we're leaning more towards these days).

Reply:
Good point, tidied.